documentation/EN/chapter_06.md

```diff
@@ -0,0 +1,104 @@
+# 6. Locha Mesh Hardware 
+ 
+This section presents the parameters that were used to choose all the control and transmission devices currently used in Locha Mesh. 
+
+
+## 6.1 Radio frequency module 
+
+<img src="imple_pic/RF-selection.svg"  height="650" width="450" align="left"/>
+
+To choose the radio module that contains the microcontroller (MCU), the RF and modulation needed to transmit on the 915 MHz frequency, some parameters were established:
+- Support of IEEE 802.15.4 standard. 
+- At least 2 USART.07 ports. 
```

Comment:
```suggestion
- At least 2 UART ports. 
```

@gustavosinbandera1 is this the correct term?

Reply:
:+1: